numba/cuda/cudadrv/driver.py

```diff
@@ -1348,7 +1645,25 @@ def own(self):
 
 
 class PinnedMemory(mviewbuf.MemAlloc):
-    def __init__(self, context, owner, pointer, size, finalizer=None):
+    """A pointer to a pinned buffer on the host.
+
+    :param context: The context in which the pointer was mapped.
+    :type context: Context
+    :param owner: The object owning the memory. For EMM plugin implementation,
+                  this ca
+    :param pointer: The address of the buffer.
+    :type pointer: ctypes.c_void_p
+    :param size: The size of the buffer in bytes.
+    :type size: int
+    :param owner: An object owning the buffer that has been pinned. For EMM
+                  plugin implementation, the default of ``None`` suffices for
+                  memory allocated in ``memhostalloc`` - for ``mempin``, it
+                  should be the owner passed in to the ``mempin`` method.
+    :param finalizer: A function that is called when the buffer is to be freed.
+    :type finalizer: function
+    """
+
+    def __init__(self, context, pointer, size, owner=None, finalizer=None):
```

Comment:
Is this an API change, any idea of prevalence of use elsewhere, does it need a deprecation cycle?

Reply:
It was an internal API before this PR, so my inclination is not to give it a deprecation cycle.

Reply:
agree